source/common/config/grpc_stream.h

```diff
@@ -60,12 +69,13 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
     }
     stream_ = async_client_->start(service_method_, *this, Http::AsyncClient::StreamOptions());
     if (stream_ == nullptr) {
-      ENVOY_LOG(warn, "Unable to establish new stream");
+      ENVOY_LOG(debug, "Unable to establish new grpc config stream");
```

Comment:
Shouldn't this be a warning? There's a real failure ocurring.

Reply:
The real failure with error message is recorded in the call back. I don;t think there is a case where something is not recorded in the callback, but *is* recorded here. This is also the source of noise since it does not tell the reason.